pkg/sink/kafka/options.go

```diff
@@ -284,21 +284,21 @@ func (o *Options) Apply(changefeedID common.ChangeFeedID,
 		o.DialTimeout = a
 	}
 
-	if urlParameter.WriteTimeout != nil && *urlParameter.WriteTimeout != "" {
-		a, err := time.ParseDuration(*urlParameter.WriteTimeout)
-		if err != nil {
-			return err
-		}
-		o.WriteTimeout = a
-	}
-
-	if urlParameter.ReadTimeout != nil && *urlParameter.ReadTimeout != "" {
-		a, err := time.ParseDuration(*urlParameter.ReadTimeout)
-		if err != nil {
-			return err
-		}
-		o.ReadTimeout = a
-	}
+	// if urlParameter.WriteTimeout != nil && *urlParameter.WriteTimeout != "" {
```

Comment:
Why comment out this part?

Reply:
confluent-kafka-go use `socket.timeout.ms` is enough.

Reply:
if this is unnecesary, just remove the code